prac_10/testing.py

```diff
@@ -0,0 +1,85 @@
+"""
+CP1404/CP5632 Practical
+Testing demo using assert and doctest
+"""
+
+import doctest
+from prac_06.car import Car
+
+
+def repeat_string(s, n):
+    """Repeat string s, n times, with spaces in between."""
+    return " ".join([s] * n)
+
+
+def is_long_word(word, length=5):
+    """
+    Determine if the word is as long or longer than the length passed in
+    >>> is_long_word("not")
+    False
+    >>> is_long_word("supercalifrag")
+    True
+    >>> is_long_word("Python")
+    True
+    """
+    return len(word) > length
```

Comment:
This still doesn't work: "as long or longer than the length passed in".
You should be checking with lengths and checking the boundary condition.